thunder/tests/test_grad.py

```diff
@@ -296,17 +296,15 @@ def check_vjp(f, *primals, comp, executor="torch", set_compile_data: bool = Fals
 
     u = tree_map(make, primals)
 
-    # dirty little trick for speed: skip the prologue
-    jf = executor.make_callable(f, disable_torch_autograd=True)
-    comp_f = thunder.compile_data(jf).get_computation_and_inputs(*primals)[0].computation_fn
+    comp_f = thunder.jit(f, disable_torch_autograd=True)
```

Comment:
@t-vi, the prologue (caching) is revived here because we now have functions that produce incorrect results when caching is skipped.
Previously caching was skipped in https://github.com/Lightning-AI/lightning-thunder/pull/1114#discussion_r1747809142.

Reply:
Yeah, so we disabled op tests on windows, it's not as critical now. In general, CI runtime is not where I would want to have it, maybe we should split op and grad...